src/components/sections/Index.tsx

```diff
@@ -0,0 +1,6 @@
+import { Hero } from "./Hero";
+
+export function Index() {
+  // put other sections here...
```

Comment:
We can treat is as an entrypoint for adding sections, because it's used by both `index.tsx` (which allows us to have default on root) and `[lang]/index.tsx`